examples/build_slab_consistent_bulk.py

```diff
@@ -0,0 +1,26 @@
+
+'''
+Testing the build_slab_consistent_bulk functionality
+'''
+
+def test_build_slab_consistent_bulk():
+    from carmm.build.slab_consistent_bulk_generator import bulk_identifier
+    from ase.io import read
+    from ase.formula import Formula
+    slab = read('data/slab_consistent_bulk/geometry_CoO_111_slab_model.in')
```

Comment:
Can we use an existing surface model in the repository? No problem to keep this, but just cautious of us introducing a new model for each test.

Reply:
I thought about that. I didn't find any existing surface model in the examples/data folder of CARMM

Reply:
You can just use the ASE slab generating functions to create one?

Reply:
See: https://wiki.fysik.dtu.dk/ase/ase/build/surface.html

Reply:
That is true. However, the whole idea of writing this script was to generate bulk where the basis vectors are transformed to generate desired surfaces (like pymatgen). ASE on the other hand does not transform the basis vectors. Hence, I believe the tests will be reliable if the checks are made against surfaces generated by pymatgen.

Reply:
I can use the pymatgen functionality to create a slab for test. This might help avoid the use of geometry file altogether

Reply:
Works for me, though you’ll have to introduce PyMatgen into the testing infrastructure (as currently it is not included). 

Also, do you need to state this is the purpose in the function preamble?

Reply:
I know you have lots of detail, but telling the user what it is supposed to work with is probably very useful.

Reply:
Yes, that makes sense. I will add a bit more detail to the docstring.

Reply:
Made the suggested changes.